pkg/adapter/cronjobevents/adapter_test.go

```diff
@@ -28,6 +28,12 @@ import (
 	"github.com/google/go-cmp/cmp"
 )
 
+type mockReporter struct{}
+
+func (r *mockReporter) ReportEventCount(args *ReportArgs, responseCode int) error {
```

Comment:
```suggestion
func (*mockReporter) ReportEventCount(args *ReportArgs, responseCode int) error {
```

Reply:
Actually we need `r`now as we are validating metric.